include/sea_dsa/DsaColor.hh

```diff
@@ -0,0 +1,138 @@
+#pragma once
+
+#include "llvm/ADT/DenseMap.h"
+#include "llvm/ADT/GraphTraits.h"
+#include "llvm/IR/CallSite.h"
+#include "llvm/IR/Function.h"
+#include "llvm/IR/Module.h"
+
+#include "sea_dsa/CallSite.hh"
+#include "sea_dsa/Global.hh"
+#include "sea_dsa/Graph.hh"
+#include "sea_dsa/GraphTraits.hh"
+
+using namespace sea_dsa;
+using namespace llvm;
+
+class Color {
+private:
+  int m_r, m_g, m_b;
+
+public:
+  Color();
+  Color(int r, int g, int b);
+
+  void randColor();
+  std::string stringColor() const;
+};
+
+enum e_color {WHITE, BLACK, GRAY}; // colors for exploration
+
+using ExplorationMap = DenseMap<const Node *, e_color>;
+using ColorMap = DenseMap<const Node *, Color>;
+using NodeSet = DenseSet<const Node *>;
```

Comment:
I think sea-dsa has already a typedef `NodeSet`

Reply:
you are right, I found `using NodeSet = boost::container::flat_set<const dsa::Node *>;` in `ShadowMem` should I use that one or give mine a different name?

Reply:
If you have the same ordering then use the one defined in ShadowMem.